test/Microsoft.ML.TestFramework/Attributes/NotArmFactAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.TestFramework.Attributes
```

Comment:
In Runtime repo we usually use ConditionalFact and we pass there the condition instead of creating a new Fact attribute for different cases. Also, we have there the PlatformDetection which we usually contain the properties for different configurations. Did we think to do something similar in machinelearning repo?

Reply:
That makes a lot of sense. We don't really have anything like that in machinelearning. The idealistic goal is that there won't be any differences between the different architectures or OS's. When that does happen, it should be the rare exception to the rule. So I think thats why we don't have anything like that. I think using a `ConditionalFact` makes a lot of sense though. Can you point me to where you do something like that in Runtime and I'll make that change.

Reply:
Here is an example:

https://github.com/dotnet/runtime/blob/f02d9ffa2ad17f69971d7774d1439ae9aa72e520/src/libraries/System.ServiceProcess.ServiceController/tests/ServiceControllerTests.cs#L33